src/boundary/Multigrid.cpp

```diff
@@ -779,62 +777,13 @@ Obstacle **Multigrid::obstacle_dominant_restriction(size_t level) {
         size = data.size();
     }
 
-    size_t *obstacle_list_tmp = new size_t[size];
+    auto *obstacle_list_tmp = new size_t[size];
```

Comment:
`auto *` looks redundant. does it need to be always a pointer or is a plain `auto` an option?

Reply:
changed in d4f2721